modules/ui/app/styles/wasabi.scss

```diff
@@ -4227,6 +4227,7 @@ form input.ng-dirty.ng-invalid:not(.ng-focused) {
     ul {
         list-style-position: inside;
         list-style-type: disc;
+        margin-left: 10px;
```

Comment:
Maybe the whitespace between the · and the text could be a little smaller (and you could probably even add some more margin here).